tools/msconvert/msconvert_macros.xml

```diff
@@ -217,14 +219,13 @@
 
       #if $general_options.multi_run_output.do_multi_run_output == 'false':
         --outfile '${os.path.splitext($basename)[0]}'
-        && sudo mv 'outputs/${os.path.splitext($basename)[0]}.${output_type}' '${output}' && sudo chown \$uid:\$gid '${output}'
+        && mv 'outputs/${os.path.splitext($basename)[0]}.${output_type}' '${output}'
       #else
-        && sudo chown \$uid:\$gid 'outputs' -R
```

Comment:
@bernt-matthias this might be a problem. Can you run this locally and check what the permissions are of the outputs after the tool has been running?

I assume they are not owned by the Galaxy user. So in production cleanup scripts, etc can not clean them?

Reply:
Will do.

Reply:
In my setup I run jobs as real user and use the option to output to the wob working dir. This dir is chowned after the run to the galaxy system user...  but I run with singularity anyway .. so this info does not help much .. but as far as I remember real user setups require output to working dir?  

For the `planemo test` run on my local machine also the permissions were correct. Probably this is because `docker run ...  --user 1000:1000` is used (1000 is me). So it seems that single user Galaxy setups are also no problem.